docs/pages/machine-id/reference/configuration.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Machine ID Configuration Reference
+description: Configuration reference for Teleport Machine ID.
+---
+
+```yaml
+# Debug enables verbose logging to stderr.
+debug: true
+
+# Address of the Teleport Auth Server (on-prem installs) or Proxy Server
+# (Cloud installs).
+auth_server: "auth.example.com:3025" (On-Prem)
+auth_server: "example.teleport.sh:443" (Cloud)
```

Comment:
I don't know how we prefer to write mutually-exclusive alternatives in our docs, but the duplicate `auth_server` entries here might cause confusion.